app/views/components/cases_cards.scala.html

```diff
@@ -28,39 +29,45 @@
+
+    @if(request.operator.manager) {
+        <h2 class="heading-medium">@messages("cases.menu.manager_tools")</h2>
         <div class="flex-container govuk-grid-row">
-            <a id="my-cases-id" class="link-text" href="my-cases">
+            <a id="manager-tools-users-id" class="link-text" href="#">
                 <div class="card">
-                    <h3 class="card-heading govuk-heading-s">@messages("operator.dashboard.classification.my-cases")</h3>
-                    <p class="card-body">
-                        @if(countCases.getOrElse("my-cases", "0") == 1){
-                        @messages("operator.dashboard.classification.my-cases.progress.singular", countCases.getOrElse("my-cases", "0"))
-                        } else {
-                        @messages("operator.dashboard.classification.my-cases.progress.plural", countCases.getOrElse("my-cases", "0"))
-                        }
+                    <h3 class="card-heading govuk-heading-s">@messages("page.title.manager_tools.users")</h3>
+                    <p class="card-body card-action">
+                        @messages("operator.dashboard.classification.manager-tools.users.p")
                     </p>
                 </div>
             </a>
-            <a id="my-referred-cases-id" class="link-text" href="@routes.MyCasesController.displayMyCases(ReferredByMeTab)">
+            <a id="manager-tools-tools-id" class="link-text" href="#">
                 <div class="card">
-                    <h3 class="card-heading govuk-heading-s">
-                    @messages("operator.dashboard.classification.my-cases.referredByMe")</h3>
-                    <p class="card-body">
-                    @if(referredCasesAssignedToMe == 1) {
-                        @messages("operator.dashboard.classification.my-cases.onReferralProgress.singular",
-                            referredCasesAssignedToMe)
-                    } else {
-                        @messages("operator.dashboard.classification.my-cases.onReferralProgress.plural",
-                            referredCasesAssignedToMe)
-                    }
+                    <h3 class="card-heading govuk-heading-s">@messages("page.title.manager_tools.tools")</h3>
+                    <p class="card-body card-action">
+                        @messages("operator.dashboard.classification.manager-tools.teams.p")
+                    </p>
+                </div>
+            </a>
+            <a id="manager-tools-keywords-id" class="link-text" href="#">
+                <div class="card">
+                    <h3 class="card-heading govuk-heading-s">@messages("page.title.manager_tools.keywords")</h3>
+                    <p class="card-body card-action">
+                        @messages("operator.dashboard.classification.manager-tools.keywords.p")
+                    </p>
+                </div>
+            </a>
+            <a id="manager-tools-reports-id" class="link-text" href="#">
```

Comment:
same as above

Reply:
The href is there as a placeholder for the route call when it is ready